project_install/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Project Management",
+    "summary": "Organize and plan your projects",
+    "author": "Onestein",
+    "website": "https://onestein.nl",
+    "category": "Project",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": [
+        "project",
+        "project_parent",
+        "project_timeline",
+        "project_task_add_very_high",
+        "project_scrum",
```

Comment:
I believe project_list,project_merge,project_task_default_stage,project_task_link might be useful and good to have for most customers